data_store/controllers/load_functions.py

```diff
@@ -188,6 +188,19 @@ def delete_existing_submission(submission_to_del: str) -> None:
     db.session.flush()
 
 
+def get_submission_by_programme_and_round(
+    programme: Programme,
```

Comment:
Tiny thing - but I guess this feels more reusable if it takes the programme_id instead of the full instance?